### ISSUE TITLE
why train loss is different from the paper ?

### ISSUE THREAD

**Issue by Johnson-yue** [XLabs-AI/x-flux#156]
https://github.com/XLabs-AI/x-flux/blob/47495425dbed499be1e8e5a6e52628b07349cba2/train_flux_deepspeed_controlnet.py#L213-L251

# In paper , x0 is noise, x1 is data 
```
x_t = t * x1 +(1-t) *x0,  it means start from x0 to x1 
loss = || (x1- x0) - pred(x_t) ||
```

![Image](https://github.com/user-attachments/assets/8b09e88d-6466-4461-a940-070db824f1cc)
 ---------------------------------------------
# In your code , x0 is noise ,x1 is data, the same as paper
```
x_t =(1-t)*x1 + t * x0  it is not equal to  x_t = t * x1 +(1-t) *x0
then the loss = || (x0 - x1) -pred(x_t) ||

```
why are you do this ??
